card.io/src/main/jni/nativeRecognizer.cpp

```diff
@@ -287,8 +296,22 @@ void setDetectedCardImage(JNIEnv* env, jobject jCardResultBitmap, IplImage* card
 
     IplImage* cardResult = cvCreateImageHeader(cvSize(bmInfo.width, bmInfo.height), IPL_DEPTH_8U, 4);
     cvSetData(cardResult, pixels, bmInfo.stride);
-
     dmz_YCbCr_to_RGB(cardY, bigCb, bigCr, &cardResult);
+
+    // blur cardnumber
```

Comment:
I agree, but I put it here for reasons:
 - the JNI code currently decides anyway which image to pass upwards via `setDetectedCardImage()`
 - letting the DMZ decide that requires more changes
 - Image mode issues, DMZ uses YCbCr internally
 - simpler & less changes
 - and: Im not an iOS developer :)

So what do we do? I can ask a college on Monday if he can invest some time.